pkg/apis/forklift/v1beta1/host.go

```diff
@@ -23,6 +23,7 @@ import (
 	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+//
```

Comment:
I was wondering about those empty comments as they are removed by `make fmt` - what editor do you use? can you set it in a way that these lines won't be added?

Reply:
not sure why its added, i'm using goland , removed it.